lib/atom-cfn-lint.js

```diff
@@ -14,7 +14,49 @@ express or implied. See the License for the specific language governing
 permissions and limitations under the License.
 */
 
-export default {
+import { CompositeDisposable } from 'atom'
+
+let helpers
+let cfnLintExecutablePath
```

Comment:
You could move these to be properties on the package object (`this.`), but variables scoped to the entire file work as well since the settings are changed for all instances.